js/notify/electron-notify.js

```diff
@@ -413,31 +413,13 @@ function showNotification(notificationObj) {
  */
 function setNotificationContents(notfWindow, notfObj) {
 
-    // Display time per notification basis.
-    let displayTime = notfObj.displayTime ? notfObj.displayTime : config.displayTime;
```

Comment:
why remove this code?  doesn't this control showing notification for 5s (when not sticky)?